packages/forms-web-app/src/dynamic-forms/options-question.js

```diff
@@ -101,10 +101,10 @@ class OptionsQuestion extends Question {
 				conditionalField.value = journey.response.answers[conditionalField.fieldName] || '';
 
 				optionData.conditional = {
-					html: nunjucks.render(
-						`./dynamic-components/conditional/${conditionalField.type}.njk`,
-						conditionalField
-					)
+					html: nunjucks.render(`./dynamic-components/conditional/${conditionalField.type}.njk`, {
+						...conditionalField,
+						...customViewData
```

Comment:
what does the customviewdata provide here - is it the errors?

Reply:
(I think this also causing the existing unit tests to fail)

Reply:
yep it's providing the errors. will need to fix failing tests